controllers/article-controller.js

```diff
@@ -0,0 +1,29 @@
+const { fetchArticleById } = require("../models/article-model");
+
+const getArticleById = (request, response, next) => {
+  const { article_id } = request.params;
+
+  if (/[A-Za-z]/g.test(article_id)) {
+    return response
+      .status(400)
+      .send({ msg: "Invalid id, article Id must be a number" });
+  }
+
+  fetchArticleById(article_id)
+    .then((article) => {
+      if (article) {
+        return response.status(200).send({ article });
+      }
+      if (!article) {
```

Comment:
You already appear to be handling the 404 in the model by Promise reject, so not sure why this is here?